README.md

```diff
@@ -7,32 +7,37 @@ A GitHub Action that will run [Flyway](https://flywaydb.org/) against a specifie
+| `flyway-command`              | true        | N/A     | The flyway command to run; e.g `migrate`, `validate`, etc.                                                                                                                                                                                 |
+| `migration-history-table`     | true        | N/A     | The table where the migration history lives. This is most likely dbo.MigrationHistory or Flyway.MigrationHistory.                                                                                                                          |
+| `baseline-version`            | false       | 0       | The baseline version to send to the flyway command.                                                                                                                                                                                        |
+| `managed-schemas`             | true        | N/A     | A comma separated list of schemas that are to be managed by Flyway.MigrationHistory.                                                                                                                                                       |
+| `enable-out-of-order`         | false       | false   | A switch that allows new migrations that are a lower version number than a current migration to be run.                                                                                                                                    |
+| `validate-migrations`         | false       | true    | A switch determining whether flyway should validate the migration scripts before running them.                                                                                                                                             |
+| `use-integrated-security`     | false       | false   | A switch defining whether or not to use integrated security. If not provided, a password should be.                                                                                                                                        |
+| `use-azure-managed-identity`  | false       | false   | A switch that can be used to indicate that an Azure Managed Identity should be used to authenticate with the SQL Server.                                                                                                                   |
+| `use-azure-service-principal` | false       | false   | A switch that can be used to indicate that an Azure Active Directory Service Principal will be used to authentication with the SQL Server.                                                                                                 |
+| `azure-msi-client-id`         | false       | N/A     | The Azure Client Id of the Managed Identity used to login to the database. Must be specified if the Managed Identity is a User-Assigned Managed Identity and the use-azure-managed-identity flag is set.                                   |
```

Comment:
The description indicates it must be specified if the msi is a user assigned managed identity....that kind of makes it seem like a system assigned one could be used and you don't have to provide the client id.  I assume you only have the option of using a user assigned msi?  Maybe the wording could be updated to clarify this?

Reply:
You were right, you can use a system assigned one. I can mention that it can be left off in that case to make it clear that system assigned is allowed.